Classes/Service/BackendUserService.php

```diff
@@ -63,11 +65,54 @@ public function addFieldsToUserSettings()
             ];
             $featureIds[] = $feature->getId();
         }
-        ExtensionManagementUtility::addFieldsToUserSettings(
-            '--div--;LLL:EXT:enhanced-backend/Resources/Private/Language/locallang_be.xlf:user_settings.enba.tab_label,' . implode(',', $featureIds)
-        );
+
     }
 
+    public function renderUserConfig()
+    {
+        $featureService = GeneralUtility::makeInstance(FeatureService::class);
+        $html = ['<div>'];
+        $groupId = '';
+        $groupClose = '';
+        foreach ($featureService->getAllFeatures() as $feature) {
+            if($groupId != $feature->getGroup()->getId())
+            {
+                $html[] = $groupClose.'<div class="form-group t3js-formengine-field-item">';
+                $html[] = '<h3>'.$feature->getGroup()->getTitle().'</h3>';
+                if($description = $feature->getGroup()->getDescription())
+                {
+                    $html[] = '<p>'.$description.'</p>';
+                }
+                $groupClose = '</div>';
+                $groupId = $feature->getGroup()->getId();
+            }
+            $html [] = $this->renderFeature($feature);
+        }
+        $html[] = '</div>';
+
+        return implode('', $html);
+    }
+
+    private function renderFeature(Feature $feature)
+    {
+        $html = ['<label class="enba-uc__feature">'];
+        switch ($feature->getType())
+        {
+            case 'check':
```

Comment:
Für die Types könnte man vllt constants in Feature-Service hinterlegen. Was denkst du?

Reply:
Als todo vermerken, bis wir ein Textfeld haben (aktuell nicht im sprint)